### ISSUE TITLE
Add support for typedoc 0.27

### ISSUE THREAD

**Issue by atheck** [DiFuks/typedoc-theme-hierarchy#34]
After updating typedoc to latest 0.27 version, I get the following error when running typedoc:

```txt
[error] The plugin typedoc-theme-hierarchy could not be loaded
[error] Error [ERR_REQUIRE_ESM]: require() of ES Module /node_modules/typedoc/dist/index.js from /node_modules/typedoc-theme-hierarchy/dist/index.js not supported.
Instead change the require of /node_modules/typedoc/dist/index.js in /node_modules/typedoc-theme-hierarchy/dist/index.js to a dynamic import() which is available in all CommonJS modules.
    at Object.<anonymous> (/node_modules/typedoc-theme-hierarchy/dist/index.js:4:19)
```

**Comment by DiFuks** [comment]
@atheck fixed in 5.0.4

**Comment by atheck** [comment]
Thank you very much @DiFuks
